custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_2.sql

```diff
@@ -0,0 +1,51 @@
+DROP TABLE IF EXISTS temp_agg_child_my;
+CREATE TABLE temp_agg_child_my AS 
```

Comment:
Same comment about this being faster if the temp table is distributed

Reply:
@Rohit25negi Could you explain this one?

Reply:
If you distribute the temp table across the citus workers, instead of leaving it as a single local table on the master, the queries that build it will be much quicker

Reply:
May be @calellowitz  can correct me. This table is used to fill int data to agg_Child_health table which is a local table. And since agg_child_health table is local table  it can directly use distributed table to update data in it. So I think this temp table has to be a local table.

Cal m I missing anything about the reason of making this temp table distributed? Because if this temp table is made distributed. it can't be used to update data in agg-child_health.

Reply:
Yeah, so what we do in the aggregation is to make the temp table distributed for agg level 5, then copy it after its created to the local machine for the updates on the actual table. But it made the overall update time WAY faster to do it that way. You can look at both agg_child_health and agg_awc to see examples of how we do that

Reply:
do you mean to say,
`copy from chm to distributed table + then copying from that table to local table` is quicker than `copying to local table directly like this `
@calellowitz

Reply:
yes